aiohttp_jinja2/__init__.py

```diff
@@ -151,7 +171,7 @@ async def wrapped(*args: Any) -> web.StreamResponse:
                 coro = func
```

Comment:
Is it deliberate that keyword arguments won't work here? i.e. A template() wrapped function will fail if called like `func(request=request)`.
Mypy gives an error because of this. I tried marking the overrides as positional only (with `/` in the parameters), but had to revert it due to being invalid syntax in 3.6.

If this is not deliberate, shall I just add `**kwargs` for safety?

Reply:
I'm fine with `func(request=request)` forbidding.
There are two reasons that prevent me from starting to use positional only args aggressively: backward compatibility and need to support Python 3.6-3.7